packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -391,13 +393,13 @@ export function makeGetProfilesForThread(): (state: GlobalState, props: {rootId:
     );
 }
 
-export function makeGetCommentCountForPost(): (state: GlobalState, props: {post: Post}) => number {
+export function makeGetCommentCountForPost(): (state: GlobalState, post: Post) => number {
     return createSelector(
         getAllPosts,
-        (state: GlobalState, {post}: {post: Post}) => state.entities.posts.postsInThread[post ? post.id : ''] || [],
-        (state, props) => props,
-        (posts, postsForThread, {post: currentPost}) => {
-            if (!currentPost) {
+        (state: GlobalState, post: Post) => state.entities.posts.postsInThread[post ? post.root_id || post.id : ''] || [],
+        (state, post: Post) => post,
+        (posts, postsForThread, post) => {
```

Comment:
Nothing that needs to change here, but I'm surprised that we can get away with calling this parameter `post` and then using a local variable called `post` in the `forEach` below